cypress/fixtures/b2b-mp/yves/multi-factor-authentication/dynamic-customer-mfa-auth.json

```diff
@@ -0,0 +1,27 @@
+{
+  "data": {
+    "type": "dynamic-fixtures",
+    "attributes": {
+      "operations": [
+        {
+          "type": "transfer",
+          "name": "LocaleTransfer",
+          "key": "locale",
+          "arguments": { "id_locale": 66, "locale_name": "en_US" }
+        },
+        {
+          "type": "helper",
+          "name": "haveCustomer",
+          "key": "customerOne",
```

Comment:
please check that we have the same number of customers used in tests across all demoshops: customerOne, customerTwo, customerThree